deployments/deployment.yaml

```diff
@@ -210,3 +216,6 @@ parameters:
     value: 1Gi
   - name: MEMORY_REQUESTS
     value: 100Mi
+  - name: LOGGING_LEVEL
+    value: "info"
```

Comment:
any reason to go with info instead of warn?   Testing locally, it still shows the 404s, and i'd think in a prod environment you'd just want warn, but IDK?

Reply:
thanks for this!! no reason indeed!! updating the level to warn